client/css/main.css

```diff
@@ -0,0 +1,100 @@
+body {
```

Comment:
in general use class selector instead of id selector

Reply:
Will do!